packages/default/scss/spreadsheet/_layout.scss

```diff
@@ -676,14 +678,18 @@
         .k-reset-color { border-bottom-width: 1px; }
         .k-custom-color { border-top-width: 1px; }
         .k-spreadsheet-border-type-palette {
-            width: 12.5rem;
-            height: 5rem;
-            display: flex;
-            flex-flow: row wrap;
-            align-content: space-around;
-            justify-content: space-around;
+            padding: k-map-get( $kendo-spacing, 2 );
+            display: grid;
+            grid-template-columns: repeat(5, max-content);
+            gap: k-map-get( $kendo-spacing, 2 );
+        }
+        .k-column-menu-group-header-text {
+            border-bottom-width: 0;
         }
```

Comment:
we should not override the column menu styles, thus remove this override for now